src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -127,7 +127,11 @@ function MetadataSidebarRedesign({ api, elementId, fileId, onError, isFeatureEna
     };
 
     const handleDeleteInstance = async (metadataInstance: MetadataTemplateInstance) => {
-        await handleDeleteMetadataInstance(metadataInstance);
+        try {
+            await handleDeleteMetadataInstance(metadataInstance);
+        } catch {
+            // ignore error, handled in useSidebarMetadataFetcher
+        }
```

Comment:
This fixed the `ShowErrorOnDelete` story on Safari for me.

Before this change the story was failing on Safari with all the steps passing due to uncaught error. I was able to narrow down it to this place.
Putting try catch inside `handleDeleteMetadataInstance` didn't fix the story. Put it around. We handle the errors with `onError` and `errorMessage` so no additional action is needed.